lib/grover/options_fixer.rb

```diff
@@ -0,0 +1,55 @@
+require 'grover/utils'
+
+class Grover
+
+  class OptionsFixer
+
+    FALSE_VALUES = [nil, false, 0, '0', 'f', 'F', 'false', 'FALSE', 'off', 'OFF'].freeze
+
+    def initialize(options)
+      @options = options
+    end
+
+    def run
+      fix_boolean_options!
+      fix_integer_options!
+      fix_float_options!
+      fix_array_options!
+      @options
+    end
+
+    private
+
+    def fix_boolean_options!
+      %w[display_header_footer print_background landscape prefer_css_page_size].each do |opt|
```

Comment:
I was thinking the same thing, but got lazy 😬 - and was worried it might be a little to "abstract". But your example reads really clearly. I'll put that in.